sdk/template-dpg/ci.yml

```diff
@@ -26,5 +26,5 @@ extends:
     ServiceDirectory: template-dpg
     ArtifactName: packages
```

Comment:
Could this file be copied in from somewhere else?

Reply:
Yes, it can be copied from content/ folder if customer want to enable CI